docs/source/algorithms/SANSILLReduction-v1.rst

```diff
@@ -52,6 +52,7 @@ Different input properties can be specified depending on the value of **ProcessA
 |              | * MaskedInputWorkspace        |                        |
 |              | * ReferenceInputWorkspace     |                        |
 |              | * SensitivityInputWorkspace   |                        |
+|              | * SolventInputWOrkspace       |                        |
```

Comment:
Please can you change to a lower case 'o'

Reply:
Thank you for spotting that, it is corrected now.